.vscode/.cmaketools.json

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
You forgot to remove this file from Git.

Reply:
oh good catch!,

Reply:
I think if you squash merge, it will go away (?)  (I deleted it in the recent commit)